content/help/faq/security/auth-mix-and-match.md

```diff
@@ -1,38 +1,7 @@
 ---
-title: Can I enable Istio Auth with some services while disable others in the same cluster?
+title: Can I enable Istio mutual TLS with some services while disable others in the same cluster?
```

Comment:
Can I enable mutual TLS for some services while leaving it disabled for other services in the same cluster?

Reply:
Done